tests/test_ensemble_tools.py

```diff
@@ -4,6 +4,7 @@
 from pathlib import Path
 from typing import TypedDict
 
+import numpy as np
```

Comment:
To avoid adding `numpy` as a dependency as we only use `prod` function, we could instead import
```Python
from functools import reduce
from operator import mul
```
and then use `reduce(mul, list_of_numbers_to_multiply)` to compute products.